Userland/Libraries/LibCrypto/Hash/SHA2.cpp

```diff
@@ -127,13 +127,13 @@ template<>
 #endif
 
 decltype(SHA256::transform_dispatched) SHA256::transform_dispatched = [] {
+#ifndef KERNEL
```

Comment:
Wait, what, does this break the build? `CPUFeatures::X86_SHA` etc. are not supposed to pass `is_valid_feature` constexpr check in the kernel.

https://github.com/SerenityOS/serenity/blob/4f7ac757f62962d9d0bc13de0180c7f4c2d99a28/AK/CPUFeatures.h#L18-L32